x-pack/test/functional/apps/transform/deleting.ts

```diff
@@ -0,0 +1,136 @@
+            progress: undefined,
+          },
+        },
+      },
+      {
+        suiteTitle: 'delete transform with latest configuration',
+        originalConfig: getLatestTransformConfig(PREFIX),
+        transformDescription: 'updated description',
+        transformDocsPerSecond: '1000',
+        transformFrequency: '10m',
+        expected: {
+          messageText: 'updated transform.',
+          row: {
+            status: TRANSFORM_STATE.STOPPED,
+            mode: 'batch',
+            progress: '100',
+          },
+        },
+      },
+    ];
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/ecommerce');
+      await transform.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+
+      for (const testData of testDataList) {
+        await transform.api.createAndRunTransform(
+          testData.originalConfig.id,
+          testData.originalConfig
+        );
+      }
+
+      await transform.testResources.setKibanaTimeZoneToUTC();
+      await transform.securityUI.loginAsTransformPowerUser();
+    });
+
+    after(async () => {
+      for (const testData of testDataList) {
+        await transform.testResources.deleteIndexPatternByTitle(testData.originalConfig.dest.index);
+        await transform.api.deleteIndices(testData.originalConfig.dest.index);
+      }
+      await transform.api.cleanTransformIndices();
+    });
+
+    for (const testData of testDataList) {
+      describe(`${testData.suiteTitle}`, function () {
+        it('delete transform', async () => {
+          await transform.testExecution.logTestStep('should load the home page');
+          await transform.navigation.navigateTo();
+          await transform.management.assertTransformListPageExists();
+
+          await transform.testExecution.logTestStep('should display the transforms table');
+          await transform.management.assertTransformsTableExists();
+
+          if (testData.expected.row.mode === 'continuous') {
+            await transform.testExecution.logTestStep('should have the delete action disabled');
+            await transform.table.assertTransformRowActionEnabled(
+              testData.originalConfig.id,
+              'Delete',
+              false
+            );
+
+            await transform.testExecution.logTestStep('should stop the transform');
+            await transform.table.clickTransformRowActionWithRetry(
+              testData.originalConfig.id,
+              'Stop'
+            );
+
+            await transform.testExecution.logTestStep('should display the stopped transform');
+            await transform.table.assertTransformRowFields(testData.originalConfig.id, {
+              id: testData.originalConfig.id,
+              description: testData.originalConfig.description,
+              status: testData.expected.row.status,
+              mode: testData.expected.row.mode,
+              progress: testData.expected.row.progress,
+            });
```

Comment:
I think this test step could be done outside the `if`, so we execute it for continuous and batch transforms.

Reply:
Updated here https://github.com/elastic/kibana/pull/92314/commits/58804c23861b3b5946f13c129a1a0cff0a093ab4